server/routes/lecturesRoute.js

```diff
@@ -206,4 +208,51 @@ router.get('/lectures/:lecture_id', (req, res) => {
   res.status(200).json({ id: 1, course_id: "01SQNOV" });
 });*/
 
+
+
+
+
+ /**
+ * @swagger
+ * /teachers/{teacher_id}/lectures/{lecture_id}:
+ *  delete:
+ *    tags:
+ *      - lectures
+ *    summary: "Delete a lecture"
+ *    description: "Use to request a deletion for a specific lecture"
+ *    consumes:
+ *       - "application/json"
+ *    produces:
+ *       - "application/json"
+ *    responses:
+ *       "200":
+ *         description: "Successful deletion"
+ *       "304":
+ *         description: "The lecture doesn't exist or has been deleted yet"
```

Comment:
Is it proper to return `304 Not Modified` here? I mean, actually the db status has not changed but the entire operation of canceling a lecture after the deadline has failed. Maybe a `409 Conflict` or a general error code is more suitable?